Dockerfile

```diff
@@ -201,30 +201,30 @@ COPY --from=osctl-darwin-build /osctl-darwin-amd64 /osctl-darwin-amd64
 # The kernel target is the linux kernel.
 
 FROM scratch AS kernel
-COPY --from=docker.io/autonomy/kernel:66c8150 /boot/vmlinuz /vmlinuz
-COPY --from=docker.io/autonomy/kernel:66c8150 /boot/vmlinux /vmlinux
+COPY --from=docker.io/autonomy/kernel:1e2003e /boot/vmlinuz /vmlinuz
+COPY --from=docker.io/autonomy/kernel:1e2003e /boot/vmlinux /vmlinux
 
 # The rootfs target provides the Talos rootfs.
 
 FROM build AS rootfs-base
-COPY --from=docker.io/autonomy/fhs:8467184 / /rootfs
-COPY --from=docker.io/autonomy/ca-certificates:20f39f7 / /rootfs
-COPY --from=docker.io/autonomy/containerd:03821f9 / /rootfs
-COPY --from=docker.io/autonomy/cni:063e06f / /rootfs
-COPY --from=docker.io/autonomy/dosfstools:767dee6 / /rootfs
-COPY --from=docker.io/autonomy/eudev:05186a8 / /rootfs
-COPY --from=docker.io/autonomy/iptables:a7aa58f / /rootfs
-COPY --from=docker.io/autonomy/libressl:3fca2cf / /rootfs
-COPY --from=docker.io/autonomy/libseccomp:80ea634 / /rootfs
-COPY --from=docker.io/autonomy/musl:9bc7430 / /rootfs
-COPY --from=docker.io/autonomy/runc:c79f79d / /rootfs
-COPY --from=docker.io/autonomy/socat:032c783 / /rootfs
-COPY --from=docker.io/autonomy/syslinux:85e1f9c / /rootfs
-COPY --from=docker.io/autonomy/xfsprogs:5e50579 / /rootfs
-COPY --from=docker.io/autonomy/base:f9a4941 /toolchain/lib/libblkid.* /rootfs/lib
-COPY --from=docker.io/autonomy/base:f9a4941 /toolchain/lib/libuuid.* /rootfs/lib
-COPY --from=docker.io/autonomy/base:f9a4941 /toolchain/lib/libkmod.* /rootfs/lib
-COPY --from=docker.io/autonomy/kernel:66c8150 /lib/modules /rootfs/lib/modules
+COPY --from=docker.io/autonomy/fhs:1e2003e / /rootfs
```

Comment:
Should we put this as an arg in the makefile like we do with the tools version? Unsure with the new pkgs thing if it's to be expected that all the components share the same sha.

Reply:
Unfortunately docker doesn't support using a var in `COPY --from`.